commonmark-ext-styles/src/test/java/org/commonmark/ext/styles/StylesTest.java

```diff
@@ -0,0 +1,99 @@
+package org.commonmark.ext.styles;
+
+import org.commonmark.Extension;
+import org.commonmark.parser.Parser;
+import org.commonmark.renderer.html.HtmlRenderer;
+import org.commonmark.testutil.RenderingTestCase;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.Set;
+
+public class StylesTest extends RenderingTestCase {
```

Comment:
Do you need any negative tests here? Like a check that `"This is some text with random styles afterwards {width=100px}"` just renders as text?

Reply:
Makes sense. Done.